Resources/views/full/folder.html.twig

```diff
@@ -0,0 +1,48 @@
+{% extends "eZDemoBundle::pagelayout.html.twig" %}
+
+{% block content %}
+    <section class="content-view-full">
+        <div class="class-folder">
+            <div class="row">
+                <div class="col-md-8">
+                    <div class="attribute-header">
+                        <h1>{{  ez_render_field( content, 'name' ) }}</h1>
+                    </div>
+
+                    {% for article in pagerFolder %}
+                        <section class="content-view-children">
+                            {{ render_esi( controller( 'ez_content:viewLocation', { 'locationId': article.contentInfo.mainLocationId, 'viewType': 'line' } ) ) }}
+                        </section>
+                    {% endfor %}
+
+                    {% if pagerFolder.haveToPaginate() %}
+                        <div class="pagination-centered">
+                            {{ pagerfanta( pagerFolder, 'twitter_bootstrap_translated', { 'routeName': location } ) }}
+                        </div>
+                    {% endif %}
+
+                </div>
+
+                <aside class="col-md-4">
+                    <section class="subnavigation row nav-collapse">
+                        <div class="attribute-header">
+                            <h2>{{ ez_render_field( content, 'name' ) }}</h2>
+                        </div>
+
+                        {% if subFolders|length > 0 %}
+                            <ul class="break folder-view">
+                                {% for folder in subFolders %}
+                                    {% if folder.contentInfo.mainLocationId != location.contentInfo.mainLocationId %}
```

Comment:
what is this condition for? the list of subfolders could include the parent itself or something?